pkg/filetree/search.go

```diff
@@ -98,7 +99,7 @@ func (sc searchContext) SearchByMIMEType(mimeTypes ...string) ([]file.Resolution
 		fileEntries = append(fileEntries, entries...)
 	}
 
-	refs, err := sc.referencesInTree(fileEntries)
+	refs, err := sc.firstMatchingReferences("**/*", fileEntries)
```

Comment:
There was a problem with [the alternate change](https://github.com/anchore/stereoscope/commit/b9900fbe31293c779df37409e72bf09eee4b5b5d#diff-289f1bd7c87e8eec652c5d4db11c3eae9a3ed8d3a108e13dd16d2b18dd69ce67R102) -- it was returning too many results for different layers. I'm happy to figure that out, but this change also solves the performance issue, albeit with some unnecessary glob matching.

Reply:
that's alright, the string glob match is fairly cheap when compared to other operations happening here